packages/smart-contracts/src/contracts/test/EtherFallbackPayment.sol

```diff
@@ -0,0 +1,15 @@
+// SPDX-License-Identifier: LGPL-3.0-only
+pragma solidity >=0.7.0 <0.9.0;
+
+/// @title EtherPaymentFallback - A contract that has a fallback to accept ether payments
+/// @author Richard Meissner - <richard@gnosis.pm>
+contract EtherPaymentFallback {
+    event SafeReceived(address indexed sender, uint256 value);
+
+    /// @dev Fallback function accepts Ether transactions.
+    receive() external payable {
+        emit SafeReceived(msg.sender, msg.value);
+    }
+
+    constructor() {}
+}
```

Comment:
Gnosis Safe extends this contract to allow for ether payments, no need to import the whole Gnosis Safe Contract.
(Tests fail with our previous EthFeeProxy and EthProxy).